.travis.yml

```diff
@@ -120,7 +124,7 @@ jobs:
 
       name: Standard netdata build
       script: fakeroot ./netdata-installer.sh --install $HOME --dont-wait --dont-start-it --enable-plugin-nfacct --enable-plugin-freeipmi --disable-lto
-      env: CFLAGS='-O1 -DNETDATA_INTERNAL_CHECKS=1 -DNETDATA_VERIFY_LOCKS=1'
+      env: CFLAGS="-O1 -Wall -Wextra -Wformat-signedness -fstack-protector-all -DNETDATA_INTERNAL_CHECKS=1 -D_FORTIFY_SOURCE=2 -DNETDATA_VERIFY_LOCKS=1"
```

Comment:
Hi @amoss,

Please change the CFLAGS for the unit tests as well

Reply:
Also, please keep the single quotes, unless double quotes are needed for interpolation

Reply:
We don't use those - they are not unit tests, they are a really bad idea. I think we should remove that stage completely until we are ready to switch on the CMocka unit tests - @cosmix?

Will change to single quotes.

Reply:
I agree @amoss. I think, however, that we should keep whatever is there until we can replace it. In other words, as soon as we have our initial CMocka suite for e.g. the exporting engine, replace it please. cc @vlvkobal @knatsakis

Reply:
So I'll leave it in, but ignore the CFLAGS changes in that stage as it will not have any effect. I'm ready to push the docs changes to match but there is no Travis output yet. Can we check if I have horribly broken it.

Reply:
Which is not a unit test? In any case, `-Wformat-signedness` does not work with old compilers and can break with many distributions.